charts/fluent-operator/values.yaml

```diff
@@ -59,21 +59,35 @@ input:
 containerRuntime: docker
 logPath:
   crio: /var/log
-  crictlPath: /usr/local/bin/crictl
+  containerd: /var/log
+
+flentd:
+  name: flentd
+  port: 24224
+  image: "kubesphere/fluentd"
+  tag: "v1.14.4"
+  watchedNamespaces:
+    - kube-system
+    - kubesphere-monitoring-system
+
 
 output:
   #Configure the output plugin parameter in FluentBit.
   #You can set enable to true to output logs to the specified location.
-  es:
-    enable: false
-    host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
-    port: 9200
-  forward:
-    enable: false
-    host: "<fluentd url like fluentd.kubesphere-logging-system.svc>"
-    port: 24224
-  kafka:
-    enable: false
-    brokers: "<kafka broker list like xxx.xxx.xxx.xxx:9092,yyy.yyy.yyy.yyy:9092>"
+  fluentbit:
+    forward:
```

Comment:
fluentbit also acts as a output layer to send to kafka or es. Pls fill back them.

Reply:
Thanks. Fixed.